public_suffix_list.dat

```diff
@@ -15564,6 +15564,14 @@ wmcloud.org
 panel.gg
 daemon.panel.gg
 
+// Wix.com, Inc. : https://www.wix.com
+// Submitted by Shahar Talmi <shahar@wix.com>
```

Comment:
Please use a non-personal e-mail.

Reply:
done, added both our names and changed to publicsuffixlist@wix.com